include/z64.h

```diff
@@ -740,6 +740,12 @@ typedef enum {
     /* 0x04 */ PAUSE_WORLD_MAP
 } PauseMenuPage;
 
+#define PAUSE_EQUIP_PLAYER_WIDTH 64
+#define PAUSE_EQUIP_PLAYER_HEIGHT 112
+
+#define PAUSE_EQUIP_BUFFER_SIZE sizeof(u16[PAUSE_EQUIP_PLAYER_HEIGHT][PAUSE_EQUIP_PLAYER_WIDTH])
+#define PAUSE_PLAYER_SEGMENT_GAMEPLAY_KEEP_BUFFER_SIZE 0x5000
```

Comment:
quote a long name, feel like we could maybe shorten a bit. but dont have any immediate suggestions

Reply:
yeah... to break it down
PAUSE_ because when paused and also tied to pauseCtx
PLAYER_SEGMENT_ because part of the pauseCtx->playerSegment pointed-to data
GAMEPLAY_KEEP_ because gameplay_keep
BUFFER_SIZE because not the whole gameplay_keep is kept in memory

It could be shortened to `PAUSE_PL_SEG_G_KEEP_BUF_SIZE` but... looks kinda horrible

Reply:
yeah it might not actually be an issue since its not used too much. would be fine with leaving but maybe others have suggestions so ill just leave this comment open